packages/consent/docs/README.md

```diff
@@ -158,57 +235,45 @@ with_defaults views: [:department, :small_volumes] do
 end
 ```
 
-## Permission
+### Permission
 
-A permission is what is consented to the user. It consentment to perform
-an *action* on a limited *view* of the *subject*. It marries the three concepts
-to consent an access to the user.
+Permission is what is granted to a role, or a user. It grants the ability to perform an *action*,
+on a limited scope (*view*) of the *subject*.
 
 ## CanCan Integration
 
-Consent provides a CanCan ability (Consent::Ability) to integrate your
-permissions with frameworks like Rails. To use it with Rails check out the
-example at [Ability for Other Users](https://github.com/CanCanCommunity/cancancan/wiki/Ability-for-Other-Users)
-on CanCanCan's wiki.
-
-In the ability you define the scope of the permissions. This is typically a
-user:
+Consent provides a CanCan ability (Consent::Ability) that can be initialized with a group of
+granted permissions. You can initialize a `Consent::Ability` with:
 
 ```ruby
-Consent::Ability.new(user)
+Consent::Ability.new(*context, super_user: <true|false>, apply_defaults: <true|false>, permissions: [Consent::Permission, ...])
 ```
 
-You'd more commonly define a subclass of `Consent::Ability`, and consent access
-to the user by calling `consent`:
+- `*context` is what is given to the view evaluating permission rules. That is typically a user;
+- `super_user` makes the ability to respond to `true` to any `can?` questions, and yields no
+  restrictions in any `accessible_by` and `accessible_through` queries;
+- `apply_defaults` grants actions with the `default_view` set automatically.
+- `permissions` is a collection of permissions to grant to the user
+
+### Manually consent permissions
```

Comment:
Can we add something in here about the `manage` gotcha? I know it's not something that's directly tied to consent, but it's a heavy gotcha that's a bit unintuitive.